samples/apps/copilot-chat-app/webapi/Skills/GitHubOpenApiSkill/openapi.json

```diff
@@ -0,0 +1,5935 @@
+{
```

Comment:
[nit] Could we put a README next to this file the has a link to where we got it from and if/how it's different from the original.

Reply:
Grouping all the open api schemas we add (so far just github and jira) under a single folder, Skills/OpenApiSkills/... , might be cleaner. That way we can just have a single bigger readme for them rather than multiple 2-3 line readmes

Reply:
Added new folder /Skills/OpenApiSkills/ with README